lib/ash/resource/validation/builtins.ex

```diff
@@ -197,6 +197,27 @@ defmodule Ash.Resource.Validation.Builtins do
     end
   end
 
+  @doc """
+  Validates the presence of a list of attributes.
+
+  If no options are provided, validates that they are all present.
+
+  ## Options
+
+  #{Spark.Options.docs(Ash.Resource.Validation.AttributePresent.opt_schema())}
+  """
+  @spec attribute_present(attributes :: atom | list(atom), opts :: Keyword.t()) ::
+          Validation.ref()
+  def attribute_present(attributes, opts \\ []) do
```

Comment:
Maybe we should use the plural `attributes_present` here? i.e `validate attribute_present([:foo, :bar], at_least: 1, at_most: 1)` vs `validate attribute_present([:foo, :bar], at_least: 1, at_most: 1)`, the former looks better.